java/java.api.common/src/org/netbeans/modules/java/api/common/queries/ModuleInfoAccessibilityQueryImpl.java

```diff
@@ -46,6 +49,7 @@
 import org.netbeans.api.java.source.JavaSource;
 import org.netbeans.modules.java.api.common.SourceRoots;
 import org.netbeans.modules.java.api.common.impl.MultiModule;
+import org.netbeans.spi.java.queries.AccessibilityQueryImplementation;
```

Comment:
import is probably not needed since `AccessibilityQueryImplementation2` is in use looks like